ir/src/symbol_table/mod.rs

```diff
@@ -230,7 +227,7 @@ impl SymbolTable {
         Ok(TraceAccess::new(
             columns.trace_segment(),
             columns.offset() + trace_access.col_offset(),
-            1,
+            columns.size(),
```

Comment:
Shouldn't this size should be determined by the size of the `trace_access`?

Reply:
Thanks, fixed.